src/complete_analysis_endpoint.py

```
@@ -0,0 +1,203 @@
+                return {"error": error_msg}, 400
+
+            start_time = time.time()
+
+            # Load models if not already loaded
+            if not (self.emotion_model_loaded and self.summarization_model_loaded and self.transcription_model_loaded):
+                self.load_models()
+
+            # Extract parameters
+            text = data.get('text', '').strip()
+            audio_data = data.get('audio_data', '').strip()
+            audio_format = data.get('audio_format', 'wav')
```

Ruff (0.12.2)

Local variable `audio_format` is assigned to but never used

Remove assignment to unused variable `audio_format`

(F841)